docs/docs/fan-unit/control/uart.mdx

```diff
@@ -0,0 +1,86 @@
+
+import Admonition from '@theme/Admonition';
+
+# Smart
+
+The Smart Fan unit utilizes the [RP2040](https://www.raspberrypi.com/products/rp2040/specifications/) the Raspberry Pi microcontroller featured in thr [Raspberry Pi Pico 1](https://www.raspberrypi.com/documentation/microcontrollers/pico-series.html). Using the RP2040's two UART connections a link can be established out of to the to blades using the fan connector
+
+## Pins
+
+The A port on the Fan unit uses `UART0` and the B port uses `UART1` on the RP2040. On the CM4 connects using `UART5`. both blade connectors can be used to power the Fan unit with +5v.
+
+<details>
+  <summary>Pin Out Guides</summary>
+  <h3>Smart Fan Unit</h3>
+  <img src='/img/fan-unit/smart-gpio.svg' width={'70%'} alt="Smart Fan Unit blade connecter pin out"/>
+  <h3>Compute blade</h3>
+  <img src='/img/hardware/fan-port.webp' width={'70%'} alt="Blade Fan unit connector"/>
+  
+</details>
+
+
+## Update firmware
+
+To start using the the Smart Fan Unit it first should be flashed with the latest firmware of your choice. This could be custom firmware or a preexisting framework.
+
+First start by removing power from the Fan Unit. Then hold down the the `BOOT` button and connect the USB Type-C to your computer.
+
+{/* Image of fan unit with Boot held down and usb plugged in */}
+
+The Fan Unit will now appear as "RPI-RP2" a storage device to your computer.
+
+{/* Windows file manager this PC */}
+
+Your desired firmware will have the file extension `.uf2`
+Move the `*.uf2` file into `RPI-RP2` it will then disconnect and reboot to the now installed firmware
+
+## CircuitPython
+
+[CircuitPython](https://circuitpython.org/) is a programming language designed to simplify experimenting and learning to code on low-cost microcontroller boards.
+
+### Installing CircuitPython
+1. Download the latest version of [CircuitPython 9](https://circuitpython.org/board/raspberry_pi_pico/)
+2. Move `adafruit-circuitpython-raspberry_pi_pico-en_US-*.uf2` file in to `RPI-RP2`
+4. It will be disconnected and `LED 3` will blink green
+3. The fan Unit with reconnect as `CIRCUITPY` will be connected.
+
+### Deploying code
+
+1. Having installed CircuitPython and the fan unit being connected to you computer via USB C
+2. Download the [example code](https://github.com/uptime-industries/circuitpy-fan-unit)
+3. Should be placed in `CIRCUITPY` if prompted to replace any file click yes
+
+### Enable UART on CM4
+
+To enable UART on the compute blade, open `/boot/firmware/config.txt`
+
+```bash 
+sudo nano /boot/firmware/config.txt
+```
+
+<Admonition type='info'>
+    Instead of the BIOS found on a conventional PC, Raspberry Pi devices use a configuration file called config.txt. The GPU reads config.txt before the Arm CPU and Linux initialise. Raspberry Pi OS looks for this file in the boot partition, located at /boot/firmware/.
```

Comment:
```suggestion
    Instead of the BIOS found on a conventional PC, Raspberry Pi devices use a configuration file called config.txt. The GPU reads config.txt before the ARM CPU and Linux are initialized. Raspberry Pi OS looks for this file in the boot partition, located at `/boot/firmware/`.
```